package.yaml

```diff
@@ -85,7 +85,6 @@ tests:
     main: Spec.hs
     source-dirs:
       - tests
-      - src
```

Comment:
Why?  I'm not sure, but don't we need this for sensei to pick up changes under `./src/` when re-running the test suite?